pkg/quota/admission/clusterresourceoverride/admission.go

```diff
@@ -48,9 +48,9 @@ func init() {
 }
 
 type internalConfig struct {
-	limitCPUToMemoryRatio     *inf.Dec
-	cpuRequestToLimitRatio    *inf.Dec
-	memoryRequestToLimitRatio *inf.Dec
+	limitCPUToMemoryRatio     float64
```

Comment:
The change to float64 lgtm, but it'd be good for @dobbymoodge to take a look since he wrote the plugin.